addon/utils.js

```diff
@@ -97,6 +100,79 @@ export function generateFacetView (facets) {
   }
 }
 
+/**
+ * Get merged definition for current cell
+ * @param {BunsenCell} cellConfig - current cell
+ * @param {Object<String, BunsenCell>} cellDefinitions - list of cell definitions
+ * @returns {BunsenCell} current merged cell definition
+ */
+export function getMergedConfig (cellConfig, cellDefinitions) {
+  if (!cellConfig.extends) {
+    return _.cloneDeep(cellConfig)
+  }
+
+  const superCell = getMergedConfig(cellDefinitions[cellConfig.extends], cellDefinitions)
+  const mergedConfig = assign(superCell, cellConfig)
+
+  delete mergedConfig.extends
+
+  return mergedConfig
+}
+
+export function getRendererComponentName (rendererName) {
+  return builtInRenderers[rendererName] || rendererName
+}
+
+/**
+ * Determine if last segment of a bunsen model path is required
+ * @param {String} path - path to property in bunsen model
+ * @param {BunsemModel} bunsenModel - bunsen model
+ * @returns {Boolean} whether or not last path is required
+ */
+export function isLastSegmentRequired (path, bunsenModel) {
+  const segments = path.split('.')
+  const lastSegment = segments.pop()
+
+  // Make sure we get the correct bunsen model for nested properties
+  if (segments.length !== 0) {
+    bunsenModel = get(bunsenModel, getModelPath(segments.join('.'))) || bunsenModel
+  }
```

Comment:
Fair enough, I actually plan to try and find the reason I need this or condition in the first place and cleanup code elsewhere so I can remove it.